praatio/data_classes/klattgrid.py

```diff
@@ -67,8 +69,8 @@ def getAsText(self):
         try:
             self.minTimestamp = toIntOrFloat(self.minTimestamp)
             outputTxt += "xmin = %s\nxmax = %s\n" % (
-                repr(self.minTimestamp),
-                repr(self.maxTimestamp),
+                my_math.numToStr(self.minTimestamp),
```

Comment:
While investigating this bug, I realized that klattgrid also wasn't using the utility method for saving numbers in a praat-friendly way.